File: platforms/swift/examples/scene/model/mesh.swift
import FragmentColor

let mesh = Mesh()
try mesh.addVertex(
    try Vertex.pbr([0.0, 0.5, 0.0]).set("uv0", [0.5, 1.0])
)

let model = Model(mesh, Material.pbr())
try model.mesh().addVertex(
    try Vertex([-0.5, -0.5, 0.0]).set("normal", [0.0, 0.0, 1.0]).set("uv0", [0.0, 0.0])
)

File: platforms/python/examples/scene/model/mesh.py
from fragmentcolor import Material, Mesh, Model, Renderer, Vertex

renderer = Renderer()
mesh = Mesh()
mesh.add_vertex(
    Vertex([0.0, 0.5, 0.0]).set(Vertex.NORMAL, [0.0, 0.0, 1.0]).set(Vertex.UV0, [0.5, 1.0]),
)

model = Model(mesh, Material.pbr())
model.mesh().add_vertex(
    Vertex([-0.5, -0.5, 0.0]).set(Vertex.NORMAL, [0.0, 0.0, 1.0]).set(Vertex.UV0, [0.0, 0.0]),
)